share/spice/octopart/octopart.js

```diff
@@ -50,18 +50,18 @@ function ddg_spice_octopart (api_result) {
 			sourceUrl : 'http://octopart.com/partsearch#search/requestData&q=' + api_result.request.q
```

Comment:
This is actually currently broken, can you please fix?

Should be: `http://octopart.com/search?q=`